### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

File: Home.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+:root {
+    --card-background: rgb(244, 51, 151);
+    --secondary-background: rgb(247, 247, 247);
+    --grey-color: rgb(138, 136, 136);
```
Comment (hiteshverma27) [inline Home.css:10]: use hex codes for colors

File: Home.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css" />
```
Comment (hiteshverma27) [inline Home.html:7]: add more meta tags

File: mycart/cart.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
+:root {
```
Comment (hiteshverma27) [inline mycart/cart.css:1]: use 1 css file for the general styles and specific styles in specific files

**Review by hiteshverma27 — APPROVED** [review]
left some suggestions, rest looks good

File: Home.css [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+}
+.nav-bar {
+    display: flex;
+    margin: 1rem 4rem;
+    justify-content: space-between;
+    width: 85vw;
+}
+.left-nav {
+    display: flex;
+
```
Comment (manojncsoni007) [inline Home.css:30]: remove unnecessary space

**Review by manojncsoni007 — COMMENTED** [review]
rest all looks good to me

**Pull request closed without merging**